src/pages/Cart/Content/ContentPrice/ContentPrice.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import styled from 'styled-components';
+import { color } from 'styles/color';
+import StyledButton from 'components/CommonStyled/StyledButton';
+
+const ContentPrice = ({ products }) => {
+  const PriceForPills = products.map(product => {
+    if (product.checked) {
+      return Object.values(product)[3] * product.quantity;
+    } else return null;
+  });
+
+  const TotalForPills = PriceForPills.reduce((a, b) => a + b, 0);
+
+  const ShippingFee = 2500;
```

Comment:
변하지 않는 상수값의 경우, 대문자로 컴포넌트 밖에 관리할 수 있겠습니다.
props나 state 상태관리에 의존하지 않는 변수나 함수는 컴포넌트 밖에서 관리해주세요.

```suggestion
  const SHIPPING_FEE = 2500;
```